src/login/redux/sagas/resetpassword/resetPasswordSaga.js

```diff
@@ -56,6 +57,7 @@ export function* postEmailLink() {
     yield put(resp);
     if (resp.type === "POST_RESET_PASSWORD_SUCCESS") {
       history.push(`/reset-password/email-link-sent`);
+      countDownStart();
```

Comment:
I am only able to see the count down in this flow, i.e. when I add an email address and ask for a link. When I reload the page in Safari during count down it disappears. I can see that a value has successfully been stored in local storage. 
If this is not the expected behaviour, maybe you can inform EmailLinkSent when the numbers should be visible? Perhaps countDownStart also should fire on load so that the counter renders when the page is reloaded? Or maybe a combination of both?

Reply:
I updated it, so now count down will continue running, in the event of the page geting reloaded.